test/apicoverage.sh

```diff
@@ -35,16 +35,24 @@ function knative_setup() {
   install_knative_serving
 }
 
+# Wrapper to fail_test to write apicoverage junit result file.
+# Parameters: $1 - Error message to pass onto fail_test
+function fail_apicoverage_run() {
+  export BUILD_FAILED=1
```

Comment:
Unnecessary complexity. Pass a flag to your go code specifying if a failure occurred or not.

Reply:
Changed it.